querybook/server/lib/metastore/loaders/hive_metastore_loader.py

```diff
@@ -111,6 +112,18 @@ def get_hive_metastore_table_description(hmc, db_name, table_name):
         return None
 
 
+def get_partition_keys(hive_metastore_description):
+    try:
+        if not hive_metastore_description.partitionKeys:
+            return []
+        return [
+            partition_key.name
+            for partition_key in hive_metastore_description.partitionKeys
+        ]
+    except ValueError:
```

Comment:
i don't think any of the action here would raise value error? if anything it would be AttributeError right